drivers/espi/Kconfig.it8xxx2

```diff
@@ -21,6 +21,29 @@ config ESPI_PERIPHERAL_HOST_IO
 config ESPI_PERIPHERAL_DEBUG_PORT_80
 	default y
 
+config ESPI_PERIPHERAL_EC_HOST_CMD
+	default y
+
+config ESPI_IT8XXX2_HC_H2RAM_SIZE
+	int "H2RAM space for ec host command (such as Chromebook)"
+	depends on ESPI_PERIPHERAL_EC_HOST_CMD
+	default 256
+	help
+	  The valid value for this option is 16/32/64/128/256/512/1024/2048 bytes.
+
+config ESPI_PERIPHERAL_ACPI_SHM_REGION
+	default y
+
+config ESPI_IT8XXX2_ACPI_SHM_H2RAM_SIZE
+	int "H2RAM space for ACPI shared memory region (read-only region)"
+	depends on ESPI_PERIPHERAL_ACPI_SHM_REGION
+	default 256
+	help
+	  The valid value for this option is 16/32/64/128/256/512/1024/2048 bytes.
```

Comment:
Should this be a choice to make it clearer what values are valid ?

Reply:
Done.